Cargo.toml

```diff
@@ -35,14 +39,16 @@ diff = "0.1"
 pretty_assertions = "1"
 rand = "0.8"
 wat = "1"
-wasmparser = "0.104"
-wasmprinter = "0.2"
 wasmi = "0.29"
 
 [features]
 default = ["std"]
-std = ["parity-wasm/std"]
-sign_ext = ["parity-wasm/sign_ext"]
+std = []
+# Enable to let parser parse Custom section as well - required to pass all tests
+# Disabled by default because it causes some issues when encoding back to WASM.
```

Comment:
It works `fvm-wasm-instrument` because it does not manipulate this section. It reads it and dumps it in `bytes()` function. It was also working fine for us.

As said before when porting `gas_metering` and `stack_limiter` I was referring to original `wasm-instrument`, which attempts to modify it, eg. [here](https://github.com/paritytech/wasm-instrument/blob/master/src/gas_metering/mod.rs#L313).
With such approach it didn't work for us.
After studying [WASM spec, custom section](https://webassembly.github.io/spec/core/binary/modules.html#custom-section) I thought we might ignore this section.

Reply:
Custom section supported now